securityconfig/audit.yml

```diff
@@ -0,0 +1,49 @@
+_meta:
```

Comment:
What is the migration path from prior versions?

Reply:
We currently have migration path via 6.xx -> 7.1 
There is migrate api in opendistro-1.1 branch https://github.com/opendistro-for-elasticsearch/security/blob/opendistro-1.1/src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/MigrateApiAction.java 

Security Admin can also be used to migrate the document. 
However the only migration needed will be add _meta field for Audit same as for NodesDn

Reply:
What is the migration path for the plugin itself? How prior installations that do not have `audit.yml` will be upgraded to installations that require `audit.yml`?

Reply:
I have separated that out in this PR https://github.com/opendistro-for-elasticsearch/security/pull/402